### DISCUSSION TITLE
Cannot mark as Answered

### DISCUSSION THREAD

**Discussion by rappen** [community/community#2903] [Discussions] [answered]
I realize I may very well just be blind or stupid - but as a repo owner, I cannot find any way to mark someone's reply as "Answer".

![image](https://user-images.githubusercontent.com/2572253/101670346-53f20a80-3a53-11eb-891d-c57ab3c945ca.png)

There just isn't any button/menu/reaction/anything where it could be done.

![image](https://user-images.githubusercontent.com/2572253/101670536-9582b580-3a53-11eb-9874-822c1d0df65f.png)

**Comment by identithree** [comment]
I think only question askers can mark an answer as correct, either that or its not a question/answer topic.

**Reply by bingalls** [reply]
I could not mark my answer to my own question, so `Mark Answer` button still does not work. It displays & hovers, but does not respond to the click event.

**Reply by louishamelers** [reply]
I have the same issue right now... Did you solve it or is it a server fault perhaps..?

**Reply by bingalls** [reply]
No, this is an unresolved bug. Further, I doubt that I'll find the original problem page.

**Comment by becca** [comment, marked as answer]
@brndnln is correct wrt the Discussions belonging to a `question and answer` but the permissions for marking as answer are more generous.

The Discussion must, first and foremost, be in a category that is `question and answer`
https://docs.github.com/en/free-pro-team@latest/discussions/managing-discussions-for-your-community/managing-categories-for-discussions-in-your-repository#default-categories

Repo owners can mark Discussions as answers as well as triagers, maintainers and the author of a post.

**Reply by rappen** [reply]
Thanks Becca! 😊

**Reply by innovate-support** [reply]
Not exactly true.  It has something to do with if a comment was entered in the 'Write a reply' section or in the 'Suggest an answer' section.  'Write a reply' can't be marked as answers (even though that's most likely where the final answer will be found), but 'Suggest an answer' can be marked as answers.  This basically results in the wrong poster getting credit for answers.  It's a stupid way for Github to function, but we can't be surprised when it's a product that Microsoft owns.

**Reply by XJ4v1erX** [reply]
thanks for the help:)

**Comment by mcornella** [comment]
The marked answer is not entirely correct, as said in https://github.com/github/feedback/discussions/2903#discussioncomment-378803 (albeit with a negative tone that I do not condone). It would be helpful to be able to "promote" a reply as an answer or a new reply, so it can be marked as the answer.

**Comment by AnsonH** [comment]
Somehow I cannot mark an answer even though I am the question asker. The "Got it!" button and "Mark as answer" seem to have broken and not responding as I repeatedly press on them,. I tried this in both Chrome and Firefox and it doesn't work. Is this a bug?

![20210823_120242](https://user-images.githubusercontent.com/57580593/130388610-7abd242a-b467-40cc-a42a-ef7facb2f401.gif)

Here's a link to [my question](https://github.com/gatsbyjs/gatsby/discussions/32867) if you are interested

**Reply by crowz4k** [reply]
Same here i get status code 422 [question](https://github.com/AutoMapper/AutoMapper/discussions/3724)

**Reply by ImoutoChan** [reply]
Same for me

**Reply by whistyun** [reply]
Same for me.
I think this is caused by "Got It!".
This button is embedded with the token, but as a result the tokens appear to be duplicated.

![image](https://user-images.githubusercontent.com/13712028/130456591-77499e13-1f0d-4c53-9a5f-3082a95e9ab0.png)

I made an [issues feedback](https://github.com/github/feedback/discussions/5242)

**Reply by louishamelers** [reply]
For anyone burning to mark a response as the correct answer you can download the android/ios application and mark it trough there.

**Reply by wkolowski** [reply]
Same more me, both on Brave and Firefox.

**Reply by iammukeshm** [reply]
Facing the same issue. Looks like this feature is broken in the web application. However, I was able to mark my comment as the answer when I used the Android Application for Github.

**Reply by Kyu0z** [reply]
het

**Comment by Kyu0z** [comment]
hey

**Comment by ShawnFromTexas** [comment]
Hello! Just to add a bit from my findings.

I noticed you may have to change your question in discussion category to "Q&A" so you can mark as answered. Verses the "general" category which you cannot mark as answered.

**Reply by roke-julian-lockhart** [reply]
@ShawnFromTexas, are you willing to file a feature request to make this possible? [^1]

[^1]: [`github.com/anonaddy/anonaddy/discussions/522#discussioncomment-13675905`][1]

[1]: https://github.com/anonaddy/anonaddy/discussions/522#discussioncomment-13675905
